aws/master-cft/paid-upi/cft-mas-core.json

```diff
@@ -911,7 +909,7 @@
     "Outputs": {
         "MASCloudAutomationVersion": {
             "Description": "Version of the MAS automated deployment on Cloud",
-            "Value": "1.0"
+            "Value": "2.0-alpha.1"
```

Comment:
@vaibhavkulkarniibm Lets change this version to 2.0 now in all the templates.

Reply:
Somewhere it was `1.0` and some places it was `2.0-alpha.1`, now made it to `2.0`